roles/openshift_aws/defaults/main.yml

```diff
@@ -254,22 +262,35 @@ openshift_aws_scale_group_health_check:
   type: EC2
 
 # these will be used during upgrade
-openshift_aws_master_group_config:
+openshift_aws_instance_config:
+  # The 'buildami' key is always required here.
+  buildami:
```

Comment:
I don't understand why this is here.

Reply:
it provides data for provision_instance task